test/functional/plugin/lsp_spec.lua

```diff
@@ -2108,7 +2108,7 @@ describe('LSP', function()
     end)
   end)
 
-  describe('apply_text_edits regression tests for #20116', function()
+  describe('apply_text_edits from lsp regression tests for #20116', function()
```

Comment:
this is in a `describe('lsp')` block so we generally don't repeat that info in nested describe() blocks
```suggestion
  describe('apply_text_edits regression tests for #20116', function()
```